versioned_docs/version-2.0/tutorials/rag.mdx

```diff
@@ -0,0 +1,545 @@
+```python
```

Comment:
Need to add sidebar_position to both of these files depending on the position you want it to be in (see the other files in this folder for examples)